deploy/cr.yaml

```diff
@@ -422,7 +422,7 @@ spec:
 #          prefix: ""
 
   toolkit:
-    image: perconalab/percona-server-mysql-operator:main-toolkit
+    image: percona/percona-server-mysql-operator:0.7.0-toolkit
```

Comment:
we have an official docker image for toolkit, can we use it percona/percona-toolkit ?

Reply:
OK, let's use it.